hil-test/tests/embassy_interrupt_spi_dma.rs

```diff
@@ -122,27 +122,27 @@ mod test {
         let mut spi = Spi::new_with_config(
             peripherals.SPI2,
             Config {
-                frequency: 100.kHz(),
+                frequency: 10000.kHz(),
```

Comment:
Had to speed these (well, one but I didn't bother figuring out which one, probably SPI3) up, otherwise this test failed (but only with generic-queue). I think it's a setup time regression, and we ran out of the 1ms tick used by the interrupt task. Considering we were transferring 64 bits at 100kHz, we were already close anyway.